content/concepts/introduction/protocols/overview.md

```diff
@@ -1,206 +1,110 @@
 ---
-title: "Overview"
-description: There are protocols everywhere you look when you're writing network applications, and libp2p is especially thick with them.
+title: "What is a libp2p Protocol"
+description: "libp2p protocols"
 weight: 20
 aliases:
     - "/concepts/protocols"
     - "/concepts/fundamentals/protocols"
 ---
 
-There are protocols everywhere you look when you're writing network applications, and libp2p is
-especially thick with them.
+libp2p is composed of various core abstractions, such as
+[peer identity](../core-abstractions/peers.md#peer-id),
+and [addressing](../core-abstractions/addressing.md/), and
+relies on protocols to facilitate communication between peers.
 
-The kind of protocols this article is concerned with are the ones built with libp2p itself,
-using the core libp2p abstractions like [transport](/concepts/transport), [peer identity](/concepts/peers#peer-id/), [addressing](/concepts/addressing/), and so on.
-
-Throughout this article, we'll call this kind of protocol that is built with libp2p
-a **libp2p protocol**, but you may also see them referred to as "wire protocols" or "application protocols".
-
-These are the protocols that define your application and provide its core functionality.
-
-This article will walk through some of the key [defining features of a libp2p protocol](#what-is-a-libp2p-protocol), give an overview of the [protocol negotiation process](#protocol-negotiation), and outline some of the [core libp2p protocols](#core-libp2p-protocols) that are included with libp2p and provide key functionality.
-
-## What is a libp2p protocol?
-
-A libp2p protocol has these key features:
+A libp2p protocol is a network protocol that operates within
+the libp2p network and follows certain conventions.
```

Comment:
I think the previous intro was better and did a good job, the new lines are a bit too succinct imo.

> A libp2p protocol is a network protocol that operates within the libp2p network and follows certain conventions.

This is a very circular definition. 😵‍💫

Reply:
That's fair.

This is trying to be direct and point out *exactly* what a libp2p protocol is. I simplified it.